shadow_exe_cloudflare.py

```
@@ -0,0 +1,1116 @@
+            soup = BeautifulSoup(response.content, 'html.parser')
+            
+            # استخراج معلومات الموقع
+            title = soup.find('title')
+            title_text = title.get_text() if title else 'No title found'
+            
+            meta_description = soup.find('meta', attrs={'name': 'description'})
+            description = meta_description.get('content') if meta_description else 'No description found'
+            
+            # استخراج الروابط
+            links = soup.find_all('a', href=True)
+            internal_links = [link['href'] for link in links if link['href'].startswith('/') or domain in link['href']]
```

Ruff (0.12.2)

Undefined name `domain`

(F821)